### PR TITLE
🚀Optimize svgs used in lightbox gallery.

### PR DESCRIPTION
**Pull request by sparhami**
- All icons changes should be pixel perfect replacements of the previous ones.
- Recreated prev, next, close, and gallery buttons from scratch
- Cleaned up slides button svg

### CODE REVIEW

**Comment by sparhami** [comment]
Artisanal handcrafted `<svg>`s!

File: extensions/amp-lightbox-gallery/0.1/lightbox-controls.css [hunk 1]
```diff
@@ -72,7 +72,7 @@
...
+  background-image: url('data:image/svg+xml;charset=utf-8,<svg viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg"><g fill="#fff"><rect x="3" y="3" width="6" height="8" rx="1" ry="1" /><rect x="15" y="13" width="6" height="8" rx="1" ry="1" /><rect x="11" y="3" width="10" height="8" rx="1" ry="1" /><rect x="3" y="13" width="10" height="8" rx="1" ry="1" /></g></svg>');
```
Comment (kristoferbaxter) [inline extensions/amp-lightbox-gallery/0.1/lightbox-controls.css]: Do you need to specify the version?

File: extensions/amp-lightbox-gallery/0.1/lightbox-controls.css [hunk 2]
```diff
@@ -103,7 +103,7 @@
 }
 
 .i-amphtml-lbg-button[data-action="next"]::after {
-  background-image: url('data:image/svg+xml;charset=utf-8,<svg viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs></defs><g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"><g id="arrow_right_03" fill="#ffffff"><path d="M7.75,9.75 L7.75,15.25 C7.75,15.8022847 7.30228475,16.25 6.75,16.25 C6.19771525,16.25 5.75,15.8022847 5.75,15.25 L5.75,8.75 C5.75,8.19771525 6.19771525,7.75 6.75,7.75 L13.25,7.75 C13.8022847,7.75 14.25,8.19771525 14.25,8.75 C14.25,9.30228475 13.8022847,9.75 13.25,9.75 L7.75,9.75 Z" id="Combined-Shape" transform="translate(10.000000, 12.000000) rotate(-225.000000) translate(-10.000000, -12.000000) "></path></g></g></svg>');
+  background-image: url('data:image/svg+xml;charset=utf-8,<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path  d="M10,7.4 L14.6,12 L10,16.6" fill="none" stroke="#fff" stroke-width="2px" stroke-linejoin="round" stroke-linecap="round" /></svg>');
```
Comment (kristoferbaxter) [inline extensions/amp-lightbox-gallery/0.1/lightbox-controls.css]: Nit: There are two spaces following the `path` tagname and the first attribute (`d`).

File: extensions/amp-lightbox-gallery/0.1/lightbox-controls.css [hunk 3]
```diff
@@ -113,7 +113,7 @@
 }
 
 .i-amphtml-lbg-button[data-action="prev"]::after {
-  background-image: url('data:image/svg+xml;charset=utf-8,<svg viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"><g id="arrow_left_03" fill="#ffffff"><path d="M11.7604076,9.75 L11.7604076,15.25 C11.7604076,15.8022847 11.3126924,16.25 10.7604076,16.25 C10.2081229,16.25 9.76040764,15.8022847 9.76040764,15.25 L9.76040764,8.75 C9.76040764,8.19771525 10.2081229,7.75 10.7604076,7.75 L17.2604076,7.75 C17.8126924,7.75 18.2604076,8.19771525 18.2604076,8.75 C18.2604076,9.30228475 17.8126924,9.75 17.2604076,9.75 L11.7604076,9.75 Z" id="Combined-Shape" transform="translate(14.010408, 12.000000) rotate(-45.000000) translate(-14.010408, -12.000000) "></path></g></g></svg>');
+  background-image: url('data:image/svg+xml;charset=utf-8,<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path  d="M14,7.4 L9.4,12 L14,16.6" fill="none" stroke="#fff" stroke-width="2px" stroke-linejoin="round" stroke-linecap="round" /></svg>');
```
Comment (kristoferbaxter) [inline extensions/amp-lightbox-gallery/0.1/lightbox-controls.css]: Same here with two spaces between `tagName` and `attribute`.

File: extensions/amp-lightbox-gallery/0.1/lightbox-controls.css [hunk 4]
```diff
@@ -82,7 +82,7 @@
...
+  background-image: url('data:image/svg+xml;charset=utf-8,<svg viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg"><path d="M5,3 L19,3 C20.1045695,3 21,3.8954305 21,5 L21,19 C21,20.1045695 20.1045695,21 19,21 L5,21 C3.8954305,21 3,20.1045695 3,19 L3,5 C3,3.8954305 3.8954305,3 5,3 Z M19,15 L19,5 L5,5 L5,13.0000896 L8,10.0003584 L13,15 L16,12 L19,15 Z M15.5,10 C14.6715729,10 14,9.32842712 14,8.5 C14,7.67157288 14.6715729,7 15.5,7 C16.3284271,7 17,7.67157288 17,8.5 C17,9.32842712 16.3284271,10 15.5,10 Z" stroke="none" fill="#fff" /></svg>');
```
Comment (kristoferbaxter) [inline extensions/amp-lightbox-gallery/0.1/lightbox-controls.css]: Same here for specifying version.

Comment (kristoferbaxter) [reply extensions/amp-lightbox-gallery/0.1/lightbox-controls.css]: I ran this new SVG thru SVGOMG and saw a slightly smaller output:

Before
```

<svg viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg">
  <path d="M5,3 L19,3 C20.1045695,3 21,3.8954305 21,5 L21,19 C21,20.1045695 20.1045695,21 19,21 L5,21 C3.8954305,21 3,20.1045695 3,19 L3,5 C3,3.8954305 3.8954305,3 5,3 Z M19,15 L19,5 L5,5 L5,13.0000896 L8,10.0003584 L13,15 L16,12 L19,15 Z M15.5,10 C14.6715729,10 14,9.32842712 14,8.5 C14,7.67157288 14.6715729,7 15.5,7 C16.3284271,7 17,7.67157288 17,8.5 C17,9.32842712 16.3284271,10 15.5,10 Z" stroke="none" fill="#fff" />
</svg>
```

After
```
<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
  <path d="M5 3h14c1.1046 0 2 .8954 2 2v14c0 1.1046-.8954 2-2 2H5c-1.1046 0-2-.8954-2-2V5c0-1.1046.8954-2 2-2zm14 12V5H5v8l3-2.9996L13 15l3-3 3 3zm-3.5-5c-.8284 0-1.5-.6716-1.5-1.5S14.6716 7 15.5 7s1.5.6716 1.5 1.5-.6716 1.5-1.5 1.5z" fill="#fff"/>
</svg>
```

File: extensions/amp-lightbox-gallery/0.1/lightbox-controls.css (hunk 2 above)
Comment (kristoferbaxter) [reply extensions/amp-lightbox-gallery/0.1/lightbox-controls.css]: Ran this thru SVGOMG and it's new output was a tad smaller:

Before
```xhtml
<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
  <path d="M10,7.4 L14.6,12 L10,16.6" fill="none" stroke="#fff" stroke-width="2px" stroke-linejoin="round" stroke-linecap="round" />
</svg>
```

After
```xhtml
<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
  <path d="M10 7.4l4.6 4.6-4.6 4.6" fill="none" stroke="#fff" stroke-width="2" stroke-linejoin="round" stroke-linecap="round"/>
</svg>
```

**Pull request merged**